### DISCUSSION TITLE
Different wallets accept the same address

### DISCUSSION THREAD

**Discussion by lemon-Star** [Chia-Network/chia-blockchain#6092] [Q&A]
I set up two wallet addresses, but I see why the receiving address on the farm is the same, on the same machine

**Comment by tiny0little** [comment]
when you say "I set up two wallet addresses", do you mean you have two keys?

**Comment by lemon-Star** [comment]
yes！I used the same computer to set up two wallets, but the acceptance address was the same！



Of course I have changed it now！

The duplicate address is:： xch1ux4g8ggfqs23pa6rfrea43g00hs38wwfa8qtzjymx2xl5wyes4msvspnmr

**Reply by tiny0little** [reply]
could you post ```chia keys show | grep "First wallet address"``` output

**Reply by lemon-Star** [reply]
sorry，i do not know how to do it！
Now I've reinstalled it on a different computer

**Comment by DrakeFS** [comment]
You only farm to one address per machine.  The address is set in the config.yaml which is why, even with 2 private keys on the same machine, you will be farming to only one of them.
